### DISCUSSION TITLE
PARTNER

### DISCUSSION THREAD

**Discussion by Targetium** [community/community#123029] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Show & Tell

### Body

We have partnered with gamepad and we want to show this partnership on github, how do I do it?

**Comment by KrlitosForever** [comment, marked as answer]
Do you want it to look like this?

![image](https://github.com/community/community/assets/7423579/938d6c67-f226-4d60-b9a0-d11469cc1e3a)

If it's an organization, you should go to 'People,' search for your NIC, and change the private to public.

![image](https://github.com/community/community/assets/7423579/0d75436c-70d0-4e29-89e2-ef444e8ecd2e)

**Reply by Targetium** [reply]
I think I couldn't explain it, I want to add it as a gamepad partner, how can I do it?

**Comment by KrlitosForever** [comment]
If you are in an organization and they have different groups.
<img width="1054" alt="image" src="https://github.com/community/community/assets/7423579/21798788-05a2-4e42-888f-8e0e75b84c9c">

1. teams
2. select your team

<img width="1204" alt="image" src="https://github.com/community/community/assets/7423579/9629ff65-e688-4a5d-b43f-2f621b3fcbf4">
1. Then add user.

Another option is:
1. People
2. Invite member
<img width="1163" alt="image" src="https://github.com/community/community/assets/7423579/d3ef1d49-4f72-4a32-9f73-35b31a0d57d1">
With their email address or GitHub NIC.

I'll remain attentive
